elasticdl/python/worker/worker.py

```diff
@@ -61,8 +63,11 @@ def __init__(
             model_def: The import path to the model definition
                 function/class in the model zoo, e.g.
                 "cifar10_subclass.CustomModel".
-            model_params: The dictionary of model parameters in a string that
-                will be used to instantiate the model,
+            model_params: The dictionary of model parameters in a string
+                separated by semi-colon used to instantiate the model,
+                e.g. "param1=1,param2=2".
+            data_reader_params: The data reader parameters in a string
+                separated by semi-colon used to instantiate the data reader,
                 e.g. "param1=1,param2=2".
```

Comment:
Here we still use comma, but we use colon in other files..

Reply:
Fixed.